.storybook/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This was required to fix the storybook `React is not defined` error we ran into here: https://github.com/curology/radiance-ui/pull/319#issuecomment-679249404